src/utils/index.ts

```diff
@@ -221,6 +221,52 @@ export function getEtherscanLink(
   }
 }
 
+const provider = () => {
+  if (window.xfi && window.xfi.ethereum) {
+    return window.xfi.ethereum;
+  } else if (window.bitkeep && window.isBitKeep) {
+    return window.bitkeep.ethereum;
+  }
+  return window.ethereum;
+};
```

Comment:
can we please move this under `changeNetwork` as this is not need to be outside i think ?